src/main/java/org/cryptomator/ui/quit/QuitController.java

```diff
@@ -81,11 +88,8 @@ public void lockAndQuit() {
 			LOG.warn("Locking failed", lockAllTask.getException());
 			lockAndQuitButton.setDisable(false);
 			lockAndQuitButton.setContentDisplay(ContentDisplay.TEXT_ONLY);
```

Comment:
Since we are switching the scene anyway, we don't have change any ui elements in this scene. Hence, those lines can be removed.